components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -170,7 +175,7 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
         if (!isMobile) {
             subMenuContent = (
                 <ul
-                    className={classNames(['a11y__popup Menu dropdown-menu SubMenu', {styleSelectableItem}])}
+                    className={classNames(['a11y__popup Menu dropdown-menu SubMenu', {styleSelectableItem}, subMenuClass])}
```

Comment:
From the docs for [classNames](https://www.npmjs.com/package/classnames), the `{styleSelectableItem}` doesn't do anything. I haven't changed it. Just highlighting what seems to be dead code.

Reply:
Yep, but since `styleSelectableItem` is a boolean, it just adds the class `.styleSelectableItem` which isn't used anywhere. 

The prop is correctly used in this line https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/menu/menu_items/submenu_item.tsx#L149

The way it's used here doesn't do anything.

Reply:
I see a class named `styleSelectableItem` in `menu.scss` https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/menu/menu.scss#L107,
isn't it applied and used here?

Reply:
Ah crap! I didn't think to look in the css files. That clears it up, thanks!